vmm/src/acpi.rs

```diff
@@ -105,23 +124,252 @@ pub fn create_dsdt_table(
     dsdt
```

Comment:
You (I) forgot to add checksum like:
`dsdt.update_checksum();`

This error slit from CI because checksum verification was ignored in early stage of kernel boot (see the message below). But it will definitely make problem later.

(From kernel boot messages:)
`[    0.000000] ACPI: Early table checksum verification disabled`

Reply:
`.append_slice()` updates the checksum. If you force the checksum check on `acpi_force_table_verification` on the kernel command line it doesn't complain about it being invalid.

Reply:
Aha, yes we don't need another `update_checksum()` here. Thanks.